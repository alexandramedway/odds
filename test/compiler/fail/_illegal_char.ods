do x = 42
do illegal = ~
