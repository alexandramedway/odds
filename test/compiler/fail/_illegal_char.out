Fatal error: exception Scanner.Illegal_Character("Illegal Character '~'")
